src/pe_reports/helpers/bulletin/bulletin_generator.py

```diff
@@ -0,0 +1,274 @@
+"""
+Bulletin/bulletin_generator: A tool for creating Posture & Exposure Bulletins.
+
+Usage:
+  bulletin_generator INTEL_ITEM_ID
+
+Options:
+  -h --help                         Show this message.
+  INTEL_ITEM_ID                     Cyber Sixgill intel item id
+"""
+
+# Standard Python Libraries
+import datetime
+import logging
+import os
+
+# Third-Party Libraries
+from docopt import docopt
+import jinja2
+import pandas as pd
+import pdfkit
+
+# cisagov Libraries
+from pe_reports.data.db_query import connect
+from pe_source.data.pe_db.config import cybersix_token
+from pe_source.data.sixgill.api import intel_post
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_post(id):
+    """Retrieve a cybersix post based on the intel item id."""
+    query = f"_id:{id}"
+    token = cybersix_token()
+    resp = intel_post(token, query, frm=0, scroll=False, result_size=1)
+    return resp
+
+
+def date_format(value, format="%m/%d/%Y"):
+    """Format a date field."""
+    val = datetime.datetime.strptime(value, "%Y-%m-%dT%H:%M:%S")
+    return val.strftime(format)
+
+
+def html_builder(text):
+    """Build out an html string for command line usage."""
```

Comment:
We always encourage proper capitalization whenever possible.
```suggestion
    """Build out an HTML string for command line usage."""
```